src/jobs/docker-publish.yml

```diff
@@ -76,7 +76,14 @@ parameters:
       Include any additional steps to collect dependency information
       before `jfrog rt bp` is executed.
 
-executor: machine
+  docker-layer-caching:
+    type: boolean
+    default: false
+    description: "Should Docker Layer Caching be enabled for this executor"
+
+executor:
+  <<: machine
+  docker_layer_caching: <<parameters.docker-layer-caching>>
```

Comment:
To allow you to pass this parameter to the executor, you'll need to declare it as a parameter for the executor in `src/executors/machine.yml` and consume it in the executor definition. You can just copy the same parameter definition you added to this job into the executor.

Reply:
Yeah, makes sense 👍